1-exercises/B-highlight-words/script.js

```diff
@@ -1,5 +1,37 @@
 function highlightWords(paragraph, colours) {
-  // Write your code here...
+  //paragraph
+  const paragraphElem = document.createElement('p');
+  const wordArray = paragraph.split(' ');
+  for (let word of wordArray) {
+    let span = document.createElement('span');
+    span.innerText = `${word} `;
+    span.addEventListener('click', function(){
+      //span.style.backgroundColor = selectElem.value === 'none' ? 'initial' :  selectElem.value;
+      selectElem.value === 'none' ? span.style = "" :  span.style.backgroundColor = selectElem.value;
+    });
```

Comment:
Instead of using a separate 'if' statement to handle "none" is there something else we can use? e.g. map?